definitions/variable/technology/technologies.yaml

```diff
@@ -76,13 +76,45 @@
     unit: [EUR_2020/pkm/yr, USD_2010/pkm/yr]
     skip-region-aggregation: true
 - Variable Cost|Heat|Residential and Commercial|{Fuel}:
-    description: Capital cost for {Fuel}
+    description: Variable cost for {Fuel}
     unit: [EUR_2020/MWh, USD_2010/MWh]
     skip-region-aggregation: true
 - Variable Cost|Heat|Industrial|{Fuel}:
-    description: Capital cost for {Fuel}
+    description: Variable cost for {Fuel}
+    unit: [EUR_2020/MWh, USD_2010/MWh]
+    skip-region-aggregation: true
+- Variable Cost (incl. Fuel Cost)|Electricity|{Electricity Input}:
+    description: Variable cost for {Electricity Input}, including the fuel cost
+    unit: [EUR_2020/MWh, USD_2010/MWh]
+    skip-region-aggregation: true
+- Variable Cost (excl. Fuel Cost)|Electricity|{Electricity Input}:
+    description: Variable cost for {Electricity Input}, without the fuel cost
     unit: [EUR_2020/MWh, USD_2010/MWh]
     skip-region-aggregation: true
+- Levelized Cost|Capex|Electricity|{Electricity Input}:
```

Comment:
Is this really different from the existing variable "Capital Cost|Electricity|..."?

Reply:
Yes, this is indeed different, as @sandrinecharousset says. For capital costs, we list the actual investment costs in M€/GW, whereas within the levelized cost calculations for the technology, they are essentially converted to a "per energy produced" basis.

Reply:
thanks @loeffko for the 2 comments
@danielhuppmann seems we are all ok now, so this PR can be merged, right?